app/controllers/journals_controller.rb

```diff
@@ -1,2 +1,38 @@
 class JournalsController < ApplicationController
+  load_and_authorize_resource :user
+  load_and_authorize_resource :journal, through: :user
+
+  def index
+    @user = current_user
+  end
```

Comment:
I think you can access current user by from ApplicationController by just calling current_user.
Since this is a JournalsController, the index method should be loading all Journals.

Reply:
I just removed the entire function, but I will need to implement it later because a user with the 'admin_role' should be able to see all journals the users with the 'user_role' created.